tests/integration/components/frost-bunsen-form/renderers/datetime-test.js

```diff
@@ -150,84 +154,6 @@ describe('Integration: Component / frost-bunsen-form / renderer / datetime', fun
     })
   })
 
-  describe('when form explicitly enabled', function () {
```

Comment:
Why did you delete all of these tests? It is important to make sure the renderer honors all of these various scenarios.